File: src/main/java/io/github/chindeaone/collectiontracker/collections/prices/NpcPrices.java
package io.github.chindeaone.collectiontracker.collections.prices;

import java.util.HashMap;
import java.util.Map;

public class NpcPrices {

    public static final Map<String, Integer> collectionPrices = new HashMap<>();

    public static int getNpcPrice(String collection) {
        return collectionPrices.getOrDefault(collection, -1);
    }
}

File: src/main/java/io/github/chindeaone/collectiontracker/collections/prices/NpcPrices.kt
package io.github.chindeaone.collectiontracker.collections.prices

object NpcPrices {
    val collectionPrices: MutableMap<String, Int> = mutableMapOf()

    fun getNpcPrice(collection: String): Int = collectionPrices.getOrDefault(collection, -1)
}